bin/releasenotes.sh

```diff
@@ -83,7 +83,7 @@ do
     # Get the actual release notes lines for the selected category, exclude category names
     entry_notes_array=( `sed -n "/$entry_upper/,/###/{//!p;}" $RELEASENOTES_TEMPTXT | sed '/^$/d'` )
     # Plugin release notes use ### and distro release notes use ## for category
-    entry_upper=`echo $entry_upper | sed -E 's/[# ]*//g;s/^/## /g'`
+    entry_upper=`echo $entry_upper | sed -E 's/^#//g'`
```

Comment:
Is this code just stripping off the leading # characters? It might be helpful to rephrase the comment on line 85 to say what the code is doing, as well as why.

Reply:
Done